src/mount/mastercomm.cc

```diff
@@ -2213,6 +2223,13 @@ uint8_t fs_sauwritechunk(uint32_t inode, uint32_t chunkIndex, uint32_t &lockId,
 		setDisconnect(true);
 		return SAUNAFS_ERROR_IO;
 	}
+	for (auto& server : chunkservers) {
+		// If 127.0.0.1, let's assume it's the same as master
+		if (server.address.ip == localhost) {
+			server.address.ip=masterip;
```

Comment:
Consider adding a space in this line to both sides of the `=` .

Reply:
Done